extract_id_title_url.py

```diff
@@ -19,7 +19,10 @@
 import re
 import socket
 import sys
-import urllib2
+try:
+    from urllib.request import urlopen, Request
+except ImportError:
+    from urllib2 import urlopen, Request
```

Comment:
Could you put import on separate lines for each module.

Reply:
I didn't get what you meant. :/

Reply:
```python
try:
    from urllib.request import urlopen
    from urllib.request import Request
except ImportError:
    from urllib2 import urlopen
    from urllib2 import Request
```

Reply:
ooops I even meant

```python
try:
    from urllib.request import Request
    from urllib.request import urlopen
except ImportError:
    from urllib2 import Request
    from urllib2 import urlopen
```

* one line by import
* alphabetical order

Reply:
Oh, I got it. :D Done!